apps/etterlatte-samordning-vedtak/src/main/kotlin/no/nav/etterlatte/samordning/vedtak/SamordningVedtakRoute.kt

```diff
@@ -22,56 +24,60 @@ fun Route.samordningVedtakRoute(samordningVedtakService: SamordningVedtakService
         }
 
         get("{vedtakId}") {
-            val vedtakId = requireNotNull(call.parameters["vedtakId"]).toLong()
-
-            val tpnummer =
-                call.request.headers["tpnr"]
-                    ?: throw ManglerTpNrException()
-
-            val samordningVedtakDto =
-                try {
-                    samordningVedtakService.hentVedtak(
-                        vedtakId = vedtakId,
-                        MaskinportenTpContext(
-                            tpnr = Tjenestepensjonnummer(tpnummer),
-                            organisasjonsnr = call.orgNummer,
-                        ),
-                    )
-                } catch (e: IllegalArgumentException) {
-                    call.respondNullable(HttpStatusCode.BadRequest, e.message)
-                }
-
-            call.respond(samordningVedtakDto)
+            withContext(MDCContext()) {
```

Comment:
forsåvidt litt unødvendig for akkurat serverloggingen, men jeg kommer også til å legge inn logging av utgående requests